### PR TITLE
Deprecating ANT

### PR DESCRIPTION
**Pull request by BadHorseMonkey**
Just a little note in Daily and the description to say "we're done here..."

But nobody is going to see it unless they're already on GitHub.

and feel free to wordsmith this.  It's far from perfect.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+	but we can only encourage you to use the newer toolchain.
+
+	The new buildfile is written in gradle, and can be executed with "./gradlew" or "gradlew.bat"
+
+	Common tasks to run (which can be discovered with "./gradlew tasks") include "test", "run", and "shadowJar"
+
+	See this forum thread on our move and recent and upcoming changes:
+	https://kolmafia.us/threads/were-moving-from-sourceforge-to-github.26407/
+
+	Since Sourceforge will no longer be updated, this version (and this buildfile) will gradually ecome out of date.
```
[the remark this answers is not in the document]
Comment (BadHorseMonkey) [reply build.xml]: resolved by adding a 'b'.  TY!

[the remark this answers is not in the document]
Comment (BadHorseMonkey) [reply build.xml]: 'b's hate me...

**Comment by jaadams5** [comment]
I can always wordsmith but I can't convince myself there is any benefit.

People who are still on SourceForge will never see this and that is fine (but see below).

People who use Ant and GitHub will see this and be warned and that is fine.

I would like to see the file deleted if only so IDEs don't ask how to build but we can wait to do that.

Do we ask @gausie to unlock SourceForge, commit this there, and lock it again?

**Comment by jaadams5** [comment]
When we deprecate Ant there are some files in the repo which I believe are only used by Ant tasks.  We should reimplement the tasks or clean up the repo.  JSmooth and some of the Mac specific builds come to mind.

**Comment by Rinn** [comment]
I think this is fine, then whenever all the ant tasks are migrated to gradle it can just be torn out completely.

**Review by Rinn — APPROVED** [review]

**Comment by heeheehee-kolmafia** [comment]
@Rinn What bits of the ant build (that we want to keep) have no Gradle equivalent? Just the binary bundling?

**Comment by BadHorseMonkey** [comment]
> @Rinn What bits of the ant build (that we want to keep) have no Gradle equivalent? Just the binary bundling?

Not Rinn, but there’s also adocumentation task, an svn update task, and we played with a Test/coverage report viewer task. 

We can rebuild the update task with grgit or a fork of it, and we can probably stop the documentation task. 

We also import all the ant tasks and get the Java required version from ant, it should probably go in gradle.properties

**Pull request merged**